zokrates_core/src/proof_system/mod.rs

```diff
@@ -18,7 +18,12 @@ use serde::{Deserialize, Serialize};
 use std::io::{Read, Write};
 #[cfg(feature = "bellman")]
 use zokrates_field::BellmanFieldExtensions;
-use zokrates_field::Field;
+use zokrates_field::{Bls12_377Field, Bls12_381Field, Bn128Field, Field};
+
+pub trait NotBw6_761Field {}
```

Comment:
Shouldn't this happen in `zokrates_field` instead?

Reply:
This thing is a workaround and a bit messy, so I wasn't sure where it belongs.